src/reducer.js

```diff
@@ -25,7 +25,8 @@ const INITIAL_STATE = {
       emoji: "🐙",
       hand: [standard(1), standard(2), standard(3)],
       deck: [standard(4), standard(5), standard(6)],
-      field: []
+      field: [],
+      fatigue: 0
```

Comment:
I think you want to start this at `1`? Right now I don't take any damage on my first overdraw